susemanager-utils/susemanager-sls/src/modules/kiwi_info.py

```diff
@@ -171,27 +178,44 @@ def get_md5(path):
     return res
 
 
-def parse_kiwi_md5(path, compressed=False):
+def get_decompressed_md5(path, decompress_cmd):
+    res = {}
+    # pylint: disable-next=undefined-variable
+    if not __salt__["file.file_exists"](path):
+        return res
+
+    cmd = decompress_cmd + " '" + path + "' | md5sum"
```

Comment:
non-blocking, but isn't this easier to read?
```suggestion
    cmd = f"{decompress_cmd} '{path}' | md5sum"
```